packager/react-packager/src/DependencyResolver/DependencyGraph/ResolutionRequest.js

```diff
@@ -336,6 +336,8 @@ class ResolutionRequest {
         file = potentialModulePath + '.native.js';
```

Comment:
Thanks! There's so much drama around this subject that we're probably going to wait until it settles and gets shipped in node before taking any action on react native